### PR TITLE
refactor: only poll visible price ticker

### PR DESCRIPTION
**Pull request by alexbarnsley**
## Summary

https://app.clickup.com/t/862jyhd3g

## Checklist

-   [ ] I checked my UI changes against the design and there are no notable differences
-   [ ] I checked my UI changes for any responsiveness issues
-   [ ] I checked my UI changes in light AND dark mode
-   [ ] I checked my (code) changes for obvious issues, debug statements and commented code
-   [ ] I opened a corresponding card on Clickup for any remaining TODOs in my code
-   [ ] I added a short description on how to test this PR _(if necessary)_
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [ ] Ready to be merged

### CODE REVIEW

File: resources/views/livewire/price-ticker.blade.php [hunk 1]
```diff
@@ -1,10 +1,10 @@
 @php ($isDisabled = ! Network::canBeExchanged() || config('explorer.network') !== 'production' || ! $isAvailable)
 
 <div
-    wire:poll.60s
+    wire:poll.visible.60s
```
Comment (ItsANameToo) [inline resources/views/livewire/price-ticker.blade.php:4]: isn't it always visible in the navbar? 🤔 or does this aim at mobile

File: resources/views/livewire/price-ticker.blade.php [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
     class="w-full md:w-auto"
     :class="{ 'opacity-50': busy }"
-    x-data="{ to: '{{ $to }}', busy: false }"
```
Comment (ItsANameToo) [inline resources/views/livewire/price-ticker.blade.php:7]: why was this change needed

File: resources/views/livewire/price-ticker.blade.php (hunk 1 above)
Comment (alexbarnsley) [reply resources/views/livewire/price-ticker.blade.php:4]: there are 2 instances of the component, 1 for desktop and 1 for mobile. Without this change, both components poll every 60 seconds (2 requests). With this change, only the visible component is polled - either the desktop navbar or the mobile navbar (if it has been opened). Thinking about it, is it worth dropping the poll frequency for the mobile navbar dropdown since it's not open very long generally :thinking:

File: resources/views/livewire/price-ticker.blade.php (hunk 2 above)
Comment (alexbarnsley) [reply resources/views/livewire/price-ticker.blade.php:7]: `to` isn't referenced anywhere with alpine anymore. it wasn't prior to my change that I can see either

File: resources/views/livewire/price-ticker.blade.php (hunk 1 above)
Comment (ItsANameToo) [reply resources/views/livewire/price-ticker.blade.php:4]: can leave it at 60 if it's only polling when visible anyway
